docs/android/tips/leak_canary.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Leak Canary 🐤"
+sidebar_label: "Leak Canary"
```

Comment:
I am now sure how known this is, but if I was having a memory leak, this wouldn't be my first choice of header to click on

Reply:
That's the name of the library it's pretty famous in the android world. And it is setup in our debug app and it installs the 🐤 app alongside ^^